Server/MirEnvir/Map.cs

```diff
@@ -820,6 +820,35 @@ public void Process(DelayedAction action)
             }
         }
 
+         /**
+         * return the coordinates of effect coordinates within an n x n square
+         * then use GetCell() in Map.cs to retrive real objects
+         * default 3x3
+         */
+        public static List<(int X, int Y)> GetEffectiveSquare(Point location, int mapWidth, int mapHeight, int squareEdgeLength = 3)
+        {
```

Comment:
Please change this to use a List<Point> instead to keep it inline with existing code

Alternatively you could use the method "FindAllTargets" which will do the same thing but directly return a full list of target objects. Method would need to be copied from MonsterObject to PlayerObject.

Reply:
Thanks for the comment. Fixed as required.